docs/changelog.rst

```diff
@@ -11,6 +11,8 @@ for Python 3.6).
 
 New features:
 
+* Added UTF-8 encoding enforcement to imported files in `beets/beets/ui/commands.py`.
```

Comment:
To be more specific: The `--from-logfile` now parses log files using the UTF-8 encoding, regardless of the system's configured locale.

(To be clear, this doesn't have anything to do with music files themselves, for which a text encoding is irrelevant!)

Reply:
I see, I'll update the documentation to be more accurate.